packages/dx-react-grid-material-ui/src/templates/column-chooser/overlay.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover } from 'material-ui';
+
+export const Overlay = ({
+  visible, onHide,
+  children, target,
+  ...restProps
+}) => (
+  <Popover
+    open={visible}
+    anchorEl={target}
+    onRequestClose={onHide}
+    anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
```

Comment:
Maybe we should position it at center?

Reply:
The horizontal alignment is quite [good](https://storage.googleapis.com/material-design/publish/material_v_12/assets/0Bzhp5Z4wHba3WVAwRkdSdURJZWc/component-menus-usage-appbar-dropdown-xhdpi-004.webm). But not vertical)